src/polyfill.rs

```diff
@@ -238,7 +238,7 @@ impl core::fmt::Debug for AtomicBool {
 unsafe impl Send for AtomicBool {}
 #[cfg(not(bool_native))]
 unsafe impl Sync for AtomicBool {}
-#[cfg(not(bool_native))]
+#[cfg(all(not(bool_native), feature = "catch_unwind"))]
```

Comment:
The feature is not in Cargo.toml, so it's not possible to activate it.

Either way, feature-gating this is a breaking change, your previous approach of autodetecting it in build.rs was better IMO.